### ISSUE TITLE
How to use `conditionalStyleFuncs` with a column has group?

### ISSUE THREAD

**Issue by PaleNeutron** [CNFeffery/feffery-antd-components#127]
See example below:

```python
import dash
import feffery_antd_components as fac
import feffery_utils_components as fuc
import flask
import numpy as np
from dash import Dash, Input, Output, callback, dcc, html

server = flask.Flask(__name__)
app = dash.Dash(__name__, server=server)


@app.server.route("/health")
def ping():
    return "{status: ok}"


APP_BAR_HEIGHT = 70

app.layout = html.Div(
    [
        fac.AntdButton("测试按钮", type="primary", shape="round", id="test_button"),
        fac.AntdTable(
            columns=[
                {
                    "title": "示例字段1",
                    "dataIndex": "示例字段1",
                    "group": "gp1",
                    "width": "100",
                },
                {
                    "title": "示例字段2",
                    "dataIndex": "示例字段2",
                    "group": "gp2",
                    "width": "100",
                },
                {
                    "title": "示例字段3",
                    "dataIndex": "示例字段3",
                    "group": "gp2",
                    "width": "100",
                },
            ],
            data=[
                {"示例字段1": i, "示例字段2": i, "示例字段3": round(np.random.rand(), 3)}
                for i in range(10)
            ],
            id="test_table",
            bordered=True,
            conditionalStyleFuncs={
                "示例字段1": """
(record, index) => {
    if ( index % 2 === 1 ) {
        return {
            style: {
                backgroundColor: "#ebfbee"
            }
        }
    }
}
""",
                "示例字段2": """
(record, index) => {
    if ( index % 2 === 1 ) {
        return {
            style: {
                color: "red"
            }
        }
    }
}
""",
                "示例字段3": """
(record, index) => {
    if ( record['示例字段3'] <= 0.5 ) {
        return {
            style: {
                background: `linear-gradient(90deg, rgb(61, 153, 112) 0%, rgb(61, 153, 112) ${record['示例字段3']*100}%, white ${record['示例字段3']*100}%, white 100%)`,
                backgroundClip: 'padding-box'
            }
        };
    }
    return {
        style: {
            background: `linear-gradient(90deg, rgb(255, 65, 54) 0%, rgb(255, 65, 54) ${record['示例字段3']*100}%, white ${record['示例字段3']*100}%, white 100%)`,
            backgroundClip: 'padding-box'
        }
    };
}
""",
            },
        ),
    ]
)


@app.callback(Output("test_table", "data"), Input("test_button", "nClicks"))
def table_data_callback(nclicks):
    return [
        {"示例字段1": i, "示例字段2": i, "示例字段3": round(np.random.rand(), 3)}
        for i in range(100)
    ]


if __name__ == "__main__":
    app.run(debug=True, exclude_patterns=["test_space/*"])
```

It does not work, but if we remove `"group": "gp2"`, it works.

**Comment by CNFeffery** [comment]
@PaleNeutron 这个目前还没支持，我研究一下

**Comment by CNFeffery** [comment]
@PaleNeutron available from fac `0.3.0b2`:
![image](https://github.com/CNFeffery/feffery-antd-components/assets/49147660/a6c0ac1b-7cbb-44b1-aeb9-3d558637424a)

```python
import dash
from dash import html
import feffery_antd_components as fac

app = dash.Dash(__name__)

app.layout = html.Div(
    [
        fac.AntdTable(
            columns=[
                {
                    'title': '字段1-1-1',
                    'dataIndex': '字段1-1-1',
                    'group': ['字段1', '字段1-1']
                },
                {
                    'title': '字段1-1-2',
                    'dataIndex': '字段1-1-2',
                    'group': ['字段1', '字段1-1']
                },
                {
                    'title': '字段1-2',
                    'dataIndex': '字段1-2',
                    'group': '字段1'
                },
                {
                    'title': '字段2',
                    'dataIndex': '字段2'
                }
            ],
            data=[
                {
                    '字段1-1-1': 1,
                    '字段1-1-2': 1,
                    '字段1-2': 1,
                    '字段2': 1
                }
            ] * 3,
            filterOptions={
                '字段1-1-1': {}
            },
            sortOptions={
                'sortDataIndexes': ['字段1-1-1']
            },
            bordered=True,
            conditionalStyleFuncs={
                '字段1-1-1': '''
(record, index) => {
    console.log(record)
    if ( index % 2 === 1 ) {
        return {
            style: {
                backgroundColor: "#ebfbee"
            }
        }
    }
}
'''
            }
        )
    ],
    style={
        'padding': '50px 100px'
    }
)


if __name__ == '__main__':
    app.run(debug=True)

```
